src/applications/personalization/appointments/actions/index.js

```diff
@@ -164,196 +81,3 @@ export function fetchConfirmedFutureAppointmentsV2() {
     }
   };
 }
-export function fetchConfirmedFutureAppointments() {
-  return async dispatch => {
-    dispatch({
-      type: FETCH_CONFIRMED_FUTURE_APPOINTMENTS,
-    });
-
-    let facilitiesLookup = {};
-    let facilitiesResponse;
-
-    let vaAppointments = [];
-    let ccAppointments = [];
-
-    const startOfToday = moment()
-      .startOf('day')
-      .toISOString();
-
-    // Maximum number of days you can schedule an appointment in advance in VAOS
-    const endDate = moment()
-      .add(395, 'days')
-      .startOf('day')
-      .toISOString();
-
-    try {
-      const vaAppointmentsResponse = await apiRequest(
-        `/appointments?start_date=${startOfToday}&end_date=${endDate}&type=va`,
-        { apiVersion: 'vaos/v0' },
-      );
-      const ccAppointmentsResponse = await apiRequest(
-        `/appointments?start_date=${startOfToday}&end_date=${endDate}&type=cc`,
-        { apiVersion: 'vaos/v0' },
-      );
-
-      // This catches partial errors on the meta object
-      if (
-        vaAppointmentsResponse.meta?.errors?.length > 0 ||
-        ccAppointmentsResponse.meta?.errors?.length > 0
-      ) {
-        dispatch({
-          type: FETCH_CONFIRMED_FUTURE_APPOINTMENTS_FAILED,
-          errors: [
-            ...(vaAppointmentsResponse.meta?.errors || []),
-            ...(ccAppointmentsResponse.meta?.errors || []),
-          ],
-        });
-        recordEvent({
-          event: `api_call`,
-          'error-key': `server error`,
-          'api-name': 'GET appointments',
-          'api-status': 'failed',
-        });
-        return;
-      }
-
-      vaAppointments = vaAppointmentsResponse.data;
-      ccAppointments = ccAppointmentsResponse.data;
-
-      const facilityIDs = uniq(
-        vaAppointments?.map(
-          appointment =>
-            getStagingID(appointment.attributes?.sta6aid)
-              ? `vha_${getStagingID(appointment.attributes?.sta6aid)}`
-              : undefined,
-        ),
-      );
-
-      if (environment.isLocalhost() && !window.Cypress) {
-        facilitiesResponse = MOCK_FACILITIES;
-      } else {
```

Comment:
mocking the facilities automatically was such a code smell to me... I'm really glad we aren't doing this anymore and can use the mock server to simulate whatever data we want from the api.